### PR TITLE
[5.2] Docs for whereColumn function

### PR DESCRIPTION
**Pull request by srmklive**
documentation for [#13549](https://github.com/laravel/framework/pull/13549)

### CODE REVIEW

**Comment by vlakoff** [comment]
You probably meant https://github.com/laravel/framework/pull/13549 ;-)

**Comment by srmklive** [comment]
@vlakoff ouch :D .. I have corrected it now. Thanks.

File: queries.md [hunk 1]
```diff
@@ -303,6 +303,22 @@ The `whereNotNull` method verifies that the column's value is **not** `NULL`:
                         ->whereNotNull('updated_at')
                         ->get();
 
+**whereColumn**
+
+The `whereColumn` method compares the values for given columns:
+
+    $users = DB::table('users')
+                        ->whereColumn('first_name', 'last_name')
```
Comment (taylorotwell) [inline queries.md]: It's not obvious what this is doing.

**Comment by taylorotwell** [comment]
Duplicate.

**Pull request closed without merging**